src/rdma.c

```diff
@@ -143,12 +143,34 @@ static void serverRdmaError(char *err, const char *fmt, ...) {
     va_end(ap);
 }
 
+static inline int connRdmaAllowCommand(void) {
+    /* RDMA MR is not accessible in a child process, avoid segment fault due to
+     * invalid MR access, close it rather than server random crash */
+    if (server.in_fork_child != CHILD_TYPE_NONE) {
+        return C_ERR;
+    }
```

Comment:
This is a great protection! It means the IO will fail if any code tries to read or write to the RDMA client from a child process. The client will be disconnected and the server does not crash.